src/js/common/model/veteran.js

```diff
@@ -775,13 +775,18 @@ function veteranToApplication(veteran) {
       case 'cityOfBirth':
       case 'stateOfBirth':
       case 'email':
+        if (value.value === '') {
+          return undefined;
+        }
+        break;
+
       case 'homePhone':
       case 'mobilePhone':
       case 'spousePhone':
         if (value.value === '') {
           return undefined;
         }
-        break;
+        return value.value.replace(/[- )(]/g, '');
```

Comment:
Hm, thinking through if this will create any issues because the front-end strips out the same characters: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/js/common/utils/validations.js#L127.

But we're validating for `^\d{10} so hypothetically we shouldn't be allowing through any non-digit characters anyway so it should be fine?